### PR TITLE
Alasdair/ecom 2766 disable details edit for non unpublished courses

### PR DESCRIPTION
**Pull request by AlasdairSwan**
@rlucioni @jimabramson please review.

### CODE REVIEW

File: programs/static/templates/program_details.underscore [hunk 1]
```diff
@@ -32,8 +32,8 @@
             </div>
 
             <div class="js-inline-edit field">
-                <label for="program-subtitle" class="sr-only"><%= gettext('Marketing Slug') %></label>
                 <span class="js-model-value copy copy-base subtitle"><%- marketing_slug %></span>
+                <label for="program-subtitle" class="sr-only"><%= gettext('Marketing Slug') %></label>
```
Comment (jimabramson) [inline programs/static/templates/program_details.underscore]: did the moved lines in this file change behavior, or is it just a style cleanup?

Comment (AlasdairSwan) [inline programs/static/templates/program_details.underscore]: Style, I think it makes sense for the labels to be beside the corresponding inputs.

**Comment by jimabramson** [comment]
:+1:

File: programs/static/js/views/program_details_view.js [hunk 2]
```diff
@@ -39,14 +39,15 @@ define([
             },
 
             postRender: function() {
-                var courses = this.model.get('course_codes');
+                var courses = this.model.get( 'course_codes' );
 
                 _.each( courses, function( course ) {
-                    var title = course.key + 'Course';
+                    var title = course.key + 'Course',
```
Comment (rlucioni) [inline programs/static/js/views/program_details_view.js]: What's going on with this title? Should there be a space between the course key and `Course`?

Comment (AlasdairSwan) [inline programs/static/js/views/program_details_view.js]: This is actually a key for the views so ```this.courseKeyCourse``` will reference said course's view.

**Comment by rlucioni** [comment]
:+1:

**Pull request merged**